### PR TITLE
fix: missing `constants` in the rockspsec

### PR DESCRIPTION
**Pull request by spacewander**
Solve the `module 'skywalking.constants' not found` problem.
See https://github.com/apache/apisix/pull/3721/checks?check_run_id=2009896073
for the full error report.

### CODE REVIEW

**Comment by dmsolr** [comment]
Hi, @spacewander please also update `rockspec/skywalking-nginx-lua-master-0.rockspec`.

**Comment by spacewander** [comment]
Updated.

**Review by wu-sheng — APPROVED** [review]
Once merge this, I expect a release vote call.

File: rockspec/skywalking-nginx-lua-0.4-1.rockspec [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package = "skywalking-nginx-lua"
-version = "0.4-0"
+version = "0.4-1"
 source = {
    url = "git://github.com/apache/skywalking-nginx-lua",
    branch = "v0.4.0",
```
Comment (dmsolr) [inline rockspec/skywalking-nginx-lua-0.4-1.rockspec]: don't we need to update the branch?

Comment (wu-sheng) [reply rockspec/skywalking-nginx-lua-0.4-1.rockspec]: I think we need to change the tag and run another release vote call, due to the rockspec files have been managed and included by the source tar, which is changed, meaning this is another version.
```suggestion
   branch = "v0.4.1",
```

**Comment by wu-sheng** [comment]
@dmsolr Let's call a patch release vote.

**Pull request merged**